styles.css

```diff
@@ -0,0 +1,130 @@
+body {
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  font-family: "Poppins", sans-serif;
+  background-color: #e5e5e5;
+}
+
+html {
+  overflow-x: hidden;
+}
+
+h1 {
+  text-align: left;
+  margin: 149px 0 10px 26px;
+}
+
+.white {
```

Comment:
- [ ] the logo should be in left corner and the nav hamburger should be in the right corner of your header, Kindly try to stick to the design as much as possible and fix that 👍🏼 

![real](https://user-images.githubusercontent.com/80895497/153658485-0e1fa977-e273-44eb-9d0d-544d9aba07ba.PNG)